src/app/page.tsx

```diff
@@ -1,5 +1,20 @@
+"use client";
 import Image from "next/image";
+import styles from "./page.module.css";
```

Comment:
instead of importing styles explicitly and repeatedly typing className={styles.className}, you can just import the file and className="className" and it'll automatically apply the styles

Reply:
ignore this, I didn't realize what I suggested only worked for normal .css files and not .module.css files